services/dynamic-sidecar/tests/unit/test_api_containers_long_running_tasks.py

```diff
@@ -446,17 +446,18 @@ async def test_containers_down_after_starting(
 
 
 async def test_containers_down_missing_spec(
-    httpx_async_client: AsyncClient, client: Client
+    httpx_async_client: AsyncClient,
+    client: Client,
+    caplog_info_debug: LogCaptureFixture,
 ):
-    with pytest.raises(TaskClientResultError) as exec_info:
-        async with periodic_task_result(
-            client=client,
-            task_id=await _get_task_id_docker_compose_down(httpx_async_client),
-            task_timeout=CREATE_SERVICE_CONTAINERS_TIMEOUT,
-            status_poll_interval=FAST_STATUS_POLL,
-        ) as result:
-            assert result is None
-    assert 'RuntimeError("No compose-spec was found")' in f"{exec_info.value}"
+    async with periodic_task_result(
+        client=client,
+        task_id=await _get_task_id_docker_compose_down(httpx_async_client),
+        task_timeout=CREATE_SERVICE_CONTAINERS_TIMEOUT,
+        status_poll_interval=FAST_STATUS_POLL,
+    ) as result:
+        assert result is None
+    assert "No compose-spec was found" in caplog_info_debug.text
```

Comment:
I am not sure we need to check the logs unless we write something very specific like an error code. the test already checks that it does not raise which is enough right?